doc/man/man3/crypto_sign.3monocypher

```diff
@@ -138,6 +138,13 @@ if (crypto_check(signature, pk, message, 500)) {
 These functions implement PureEdDSA with Curve25519 and Blake2b, as
 described in RFC 8032.
 This is the same as Ed25519, with Blake2b instead of SHA-512.
+.Sh HISTORY
+The
+.Fn crypto_sign ,
+.Fn crypto_check ,
+and
+.Fn crypto_sign_public_key
+functions appeared in Monocypher 0.2.
```

Comment:
Perhaps we should mention the critical vulnerability in `crypto_check()`? That vulnerability was introduced in version 0.3 I believe (when I started to perform the scalar multiplication in Montgomery space), and was corrected on versions 2.0.4 and 1.1.1.

Reply:
Looking at some of the unpatched bindings that have been floating around, that might not actually be a bad idea.